relentless/core.py

```diff
@@ -1,4 +1,4 @@
-__all__ = ['Interpolator','Variable']
+__all__ = ['Interpolator','Variable','FixedKeyDict']
```

Comment:
`relentless/__init__.py` will import everything from `core.__all__` into its base namespace, so we only want to expose classes the user should actually be working with. We should omit containers like `FixedKeyDict` or `PairMatrix`.